docs/navigation-configuration.md

```diff
@@ -1,242 +1,263 @@
+Using node parameters in the previous example results in:
 
-````
-{
-  navigation: {
+```javascript
+navigation: {
     nodes: [
       {
         pathSegment: 'home',
         label: 'Home',
-        viewUrl: 'https://my.microfrontend.com/',
+        viewUrl: 'https://microfrontend.com/',
         children: [
           {
-            pathSegment: 'projects',
-            label: 'Projects',
-            viewGroup: 'projectsGroup',
-            viewUrl: 'https://my.microfrontend.com/projects/list.html#pagenr={nodeParams.page};sort={nodeParams.sorting}',
+            pathSegment: 'users',
+            label: 'User List',
+            viewUrl: 'https://microfrontend.com/users/list.html#pagenr={nodeParams.page};sort={nodeParams.sorting}',
             children: [
               {
-                pathSegment: ':projectId',
-                label: 'Project Details',
-                viewUrl: 'https://my.microfrontend.com/projects/details.html#id=:projectId;'
-              }            
-			]
+                pathSegment: ':userId',
+                label: 'User Profile',
+                viewUrl: 'https://microfrontend.com/projects/details.html#id=:userId;'
+              }
+            ]
           }
         ]
       }
     ]
   }
 } 
-
-````
+...
+```
 
  Use the following options to work with node parameters:
 
-- Build the **viewUrl** by placing them anywhere in the **viewUrl** value using the following syntax: `nodeParams.{node param name}`. For example, if the main application URL is `https://luigiexample.com/home/projects/?~sorting=asc&~page=2` then the **viewUrl** of a micro frontend is `https://my.microfrontend.com/projects/list.html#pagenr=2;sort=asc`.
-
-- Use the [Luigi Client API](luigi-client-api.md) to access the node parameter values from within the micro frontend. Use the `LuigiClient.getNodeParams()` function. 
-For example, to get the value of the sorting parameter, use `LuigiClient.getNodeParams().sorting`. 
+Build the **viewUrl** by placing parameters anywhere in the **viewUrl** value using the following syntax: `nodeParams.{node param name}`. For example, if the main application URL is `https://yourwebsite.com/home/projects/?~sorting=asc&~page=2` then the **viewUrl** of a micro frontend is `https://microfrontend.com/projects/list.html#pagenr=2;sort=asc`.
 
 
-## Dynamic viewUrl
+### Dynamic viewUrl
 
-Use the node parameters and path parameters to build a dynamic **viewUrl**.
+You can use both node parameters and path parameters to build a dynamic **viewUrl**.
 
-In this example, the web application URL is `https://Luigi.corp/something/sample_1/products?~sort=asc`. The micro frontend loads using a different URL, such as `https://admin.my.test/project/sample_1/products?sort=asc`.
+For example, if the web application URL is `https://luigi.corp/something/sample_1/products?~sort=asc`, the micro frontend will load using a different URL, such as `https://admin.my.test/project/sample_1/products?sort=asc`.
```

Comment:
```suggestion
For example, if the web application URL is `https://luigi.corp/something/sample_1/products?~sort=asc`, the micro frontend loads using a different URL, such as `https://admin.my.test/project/sample_1/products?sort=asc`.
```
I would maybe use the present tense - it's not mandatory in each and every case, but here it's the expected micro frontend behavior, I'd say.